src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -133,27 +133,46 @@ private static void Main(string[] args)
         private static void ShowResults(UpdateResults results)
         {
             Console.ForegroundColor = results.Outcome ? ConsoleColor.Green : ConsoleColor.Red;
-            Console.WriteLine($"Completed {(results.Outcome ? "successfully" : "in a failure")}.");
+            Trace.WriteLine($"Completed {(results.Outcome ? "successfully" : "in a failure")}.");
             if (!results.Outcome)
             {
-                Console.WriteLine($"Details: {results.OutcomeDetails}.");
+                Trace.WriteLine($"Details: {results.OutcomeDetails}.");
             }
             Console.ResetColor();
             Trace.WriteLine($"Duration: {results.DurationMilliseconds:N2}-ms.");
             Trace.WriteLine($"Successful updates: {results.UpdatedNuGets.Count():N0}.");
-            Trace.WriteLine("Updated default.config NuGet package versions...");
+            Trace.WriteLine("Found package version changes in config files...");
             foreach (PackageUpdateResult updatedNuget in results.UpdatedNuGets.OrderBy(r => r.PackageId))
             {
                 Trace.WriteLine($"           {updatedNuget.PackageId}: {updatedNuget.NewVersion}");
             }
 
+            Trace.WriteLine($"Updated {results.FileSaveResults?.Length ?? 0} config files...");
+            if(results.FileSaveResults != null)
+            {
+                foreach(FileSaveResult configSaveResult in results.FileSaveResults)

```

Comment:
```suggestion
                foreach(FileSaveResult configSaveResult in results.FileSaveResults ?? Enumerable.Empty<FileSaveResult>)

```

and delete the containing ```if```

Reply:
Done